lib/utils/settings.js

```diff
@@ -0,0 +1,56 @@
+import * as fs from 'fs'
+import * as os from 'os'
+import * as path from 'path'
+
+// @ts-ignore no types for ascii85
+import ascii85 from 'ascii85'
+
+let dataHome = process.platform === 'win32'
```

Comment:
I'd say just use a lib off npm instead like https://socket.dev/npm/package/xdg-portable , that will reduce maintenance burden / complexity

Reply:
I'm dissuaded by the low Socket score of all existing options + the fact this is only like 4 lines of code